### DISCUSSION TITLE
efx-toolkit-java

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#122] [Q&A] [answered]
Hi All !
anyone can tell me which version of efx-toolkit-java can I put in pom.xml ?
I put
<dependency>
			<groupId>eu.europa.ted.eforms</groupId>
			<artifactId>efx-toolkit-java</artifactId>
			<version>0.1.1</version>
		</dependency>
And it was found , but it is old,  I got the error about SdkFieldV1 class
if I write
<dependency>
			<groupId>eu.europa.ted.eforms</groupId>
			<artifactId>efx-toolkit-java</artifactId>
			<version>1.0.1</version>
		</dependency>
as in https://mavenlibs.com/maven/dependency/eu.europa.ted.eforms/efx-toolkit-java
It is not found my Maven on IDEA.
Thanks in advance
Renato

**Comment by rousso** [comment, marked as answer]
Hi, @rengit62,

1. Go to https://github.com/OP-TED/efx-toolkit-java and look at the README.md file.
2. You will find in there, at the top of the screen a link that says "Latest release artefacts". 
 This will take you to [Maven Central](https://search.maven.org/search?q=g:%22eu.europa.ted.eforms%22%20AND%20a:%22efx-toolkit-java%22) where you will see that the latest version is 1.0.1. 
3. If you follow the latest version link (1.0.1) it will take you to [this page](https://search.maven.org/artifact/eu.europa.ted.eforms/efx-toolkit-java/1.0.1/jar) where you will find everything you need to add the dependency. 
 Look at the right column and it gives you the script to add the dependency for Maven , Gradle etc.

Here is what you will find there:
```
<dependency>
  <groupId>eu.europa.ted.eforms</groupId>
  <artifactId>efx-toolkit-java</artifactId>
  <version>1.0.1</version>
</dependency>
```

That said, I visited in the link you provided and I can see that the same information also exists there:
![image](https://user-images.githubusercontent.com/91379/193265434-3b106978-5c08-484c-a7ae-244fca2d39c7.png)

Regards,
Yannis

**Reply by meletev** [reply]
@rengit62,
Hello,
It is possible that IDEA is misconfigured for Maven.
Or you might be using a proxy repository (e.g. Nexus or Artifactory) the indexes of which have not been recently updated.

Could you please try the following from the command line:
_mvn dependency:get -DgroupId=eu.europa.ted.eforms -DartifactId=efx-toolkit-java -Dversion=1.0.1 -Dpackaging=jar -DremoteRepositories=central::default::https://repo.maven.apache.org/maven2_

**Reply by rengit62** [reply]
Thanks to all !
I solved the issue , 'forcing' Maven to reload the projects
